drakcore/drakcore/frontend/src/App.js

```diff
@@ -40,26 +45,69 @@ class App extends Component {
-                    </button>
-
-                    <div className="collapse navbar-collapse" id="navbarSupportedContent">
-                        <NavBtnsWithRouter />
+            <div className="wrapper">
+                <div className="left-side-menu">
+                    <a href="/" className="logo text-center logo-light">
+                        <span className="logo-lg">
+                            <img src="/assets/images/logo.png" alt=""/>
+                        </span>
+                        <span className="logo-sm">
+                            <img src="/assets/images/logo.png" alt="" height="16"/>
+                        </span>
+                    </a>
+
+                    <div className="h-100" id="left-side-menu-container" data-simplebar>
+                        <ul className="metismenu side-nav">
+                            <li className="side-nav-title side-nav-item">Sandbox</li>
+                            <li className="side-nav-item">
+                                <Link to={"/upload"} className="side-nav-link">
+                                    <i className="uil-file-upload"/>
+                                    <span> Upload sample </span>
+                                </Link>
+                            </li>
+                            <li className="side-nav-item">
+                                <Link to={"/"} className="side-nav-link">
+                                    <i className="uil-folder"/>
+                                    <span> Analyses </span>
+                                </Link>
+                            </li>
+
+                            <NavBtnsWithRouter/>
+                        </ul>
+                        <div className="clearfix"/>
                     </div>
-                </nav>
 
-                <Switch>
-                    <Route path="/progress/:analysis" component={AnalysisStatus} />
-                    <Route path="/analysis/:analysis" component={AnalysisMain} />
-                    <Route path="/" exact component={AnalysisList} />
-                </Switch>
+                </div>
+            </div>
+
+            <div className="content-page">
+                <div className="content">
+                    <Switch>
+                        <Route path="/progress/:analysis" component={AnalysisStatus}/>
+                        <Route path="/analysis/:analysis" component={AnalysisMain}/>
+                        <Route path="/upload" exact component={UploadSample}/>
+                        <Route path="/" exact component={AnalysisList}/>
+                    </Switch>
+                </div>
             </div>
+
+            <footer className="footer">
+                <div className="container-fluid">
+                    <div className="row">
+                        <div className="col-md-6">
+                            DRAKVUF Sandbox (C) 2020 <a href="https://cert.pl/">CERT Polska</a><br/>
```

Comment:
2019-2020? :D

Reply:
it was publicly released under the name DRAKVUF Sandbox just in 2020

if we would actually count since the development was started, I guess 2019, I will change that in separate PR